apps/condo/domains/ticket/components/TicketsWidget.tsx

```diff
@@ -42,7 +42,7 @@ export const TicketsWidget = () => {
     return (
         <StatsCard
             title={TicketsWidgetTitle}
-            link='/reports/detail/report_by_tickets'
+            link='/reports/detail/report-by-tickets'
```

Comment:
Why not ```/tickets``` ?

Reply:
Because side menu active url method checks substring. So if I change path to `/tickets`, tickets section will become active state too